### DISCUSSION TITLE
Checking if this deploy request is deployable...   Hung for over 2 hours

### DISCUSSION THREAD

**Discussion by jonrcrowell** [planetscale/discussion#140] [Bug Reports] [answered]
I have two branches that are almost empty, my main branch and a branch called seedling. I'm trying to deploy a very small change and the validation is not completing.

Here are my steps:
1. I created a single table in seedling and deployed it successfully to main.
2. I dropped the table in seedling
3. I created a deployment request to main

What should I do?

**Comment by jonrcrowell** [comment]
20 hours later, it is still running. Is there a way to cancel this? Is there an error that isn't being reported to me?

**Reply by nonameolsson** [reply]
This has happened for me as well. I haven't found a way to cancel. And I even tried to delete my database. Not possible during deploy 😢  I contacted support and I'm waiting for an answer. 
This happened yesterday as well.

**Comment by mscoutermarsh** [comment, marked as answer]
This was due to a bug that was fixed, if anyone hits a similar issue in the future please reach out to support and we'll take care of it ASAP.
